unitTests/Simulate.ts

```diff
@@ -221,6 +221,31 @@ export class Simulate {
     return modalBox;
   }
 
+  static fullModalBoxModule(): ModalBox {
```

Comment:
Why did you create a new function for this instead of modifying the existing `modalBoxModule` ? It seems very hackish ;)

Reply:
I did that because the context I needed to test `Quickview` was too different from the already existing one and I didn't want to risk breaking other accessibility tests. I'm going to test this one with every accessibility test and see if it works.